package.json

```diff
@@ -4,18 +4,24 @@
   "description": "A document model for rich text editors",
   "author": "Jason Chen <jhchen7@gmail.com>",
   "homepage": "http://quilljs.com/docs/parchment",
-  "main": "./dist/parchment.js",
-  "types": "./dist/typings/parchment.d.ts",
+  "main": "./dist/parchment.umd.js",
+  "module": "./dist/parchment.mjs",
+  "types": "./dist/parchment.d.ts",
   "files": [
     "tsconfig.json",
     "dist",
     "src"
   ],
+  "overrides": {
```

Comment:
Let's fix audit issues in a separate PR and by upgrading related dependencies (ex karma-sauce-launcher).